tgui/src/components/display.ract

```diff
@@ -1,7 +1,7 @@
 <div class='display'>
   {{#if title}}
     <header>
-      <h3>{{title}}</h3>
+      <h3>{{{title}}}</h3>
```

Comment:
```{{}}``` to import text
```{{{}}}``` to import HTML/JS
If you're going to comment on moustache templates @Cyberboss, atleast understand the syntax.